library/general/src/modules/Popup.rb

```diff
@@ -74,6 +74,8 @@ def popupLayoutInternalTypeWithLabel(headline, message, button_box, label, richt
       button_box = deep_copy(button_box)
       content = Empty()
 
+      body = richtext ? message : convert_to_plaintext(message)
```

Comment:
Well, we could be more conservative and don't do any conversion at all. From my POV escaping tags when using richtext was wrong, so we could show the message as is in both cases.